recipes/openal/all/conanfile.py

```diff
@@ -20,10 +20,11 @@ class OpenALConan(ConanFile):
     _build_subfolder = "build_subfolder"
 
     def configure(self):
+        if tools.Version("1.20.1") <= self.version and self.settings.compiler.get_safe("cppstd"):
```

Comment:
I guess c++ is new from 1.21. ( a nitpick, I know)
```suggestion
        if tools.Version("1.21") <= self.version and self.settings.compiler.get_safe("cppstd"):
```